extension/src/experiments/columns/paths.ts

```diff
@@ -1,17 +1,15 @@
 import { join } from 'path'
```

Comment:
This import is what why I need to move out all constants to use them in the webview.